harness-evals/README.md

```
@@ -0,0 +1,25 @@
+# Harness evals (operator-owned)
+
+Operator-specific inputs for the [OpenSpec agile workflow](https://github.com/sujkini/openspec/blob/v2-restructured/README.md).
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)